ekglib/kgiri/namespace.py

```diff
@@ -31,11 +33,47 @@ def set_ekg_namespace_with_base(key: str, base: str, suffix: str):
 def set_kgiri_base(ekg_kgiri_base: URIRef):
     global kgiri_base
 
-    kgiri_base = ekg_kgiri_base
+    kgiri_base =  complete_ns_iri_ending(ekg_kgiri_base)
```

Comment:
Does "complete" formally mean "including a trailing slash/hash"? To me the function's name doesn't explain what it is doing

Reply:
This function existed already, it seems  to add the trailing slash if it isn't there already. Calling it to ensure consistency with the preexisting code